sdks/python/apache_beam/examples/kafkataxi/kafka_taxi.py

```diff
@@ -0,0 +1,87 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""An example that writes to and reads from Kafka.
+
+ This example reads from the PubSub NYC Taxi stream described in
+ https://github.com/googlecodelabs/cloud-dataflow-nyc-taxi-tycoon, writes to a
+ given Kafka topic and reads back from the same Kafka topic.
+ """
+
+# pytype: skip-file
+
+from __future__ import absolute_import
+
+import argparse
+import logging
+import typing
+
+import apache_beam as beam
+from apache_beam.io.kafka import ReadFromKafka
+from apache_beam.io.kafka import WriteToKafka
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+from apache_beam.options.pipeline_options import StandardOptions
+
+
+def run(argv=None):
```

Comment:
In general, for Python samples, we usually have the `run` function accept the actual values the function needs to run instead of `argv`, and we let `__main__` use `argparse` to parse the arguments.

This allows users to more easily import the sample "as a library" and run the sample from within a script. It also makes it easier to test.

It's also a good idea to show a commented example of how those arguments would look like. Ideally, if they copy-paste the contents of the `run` function and uncomment those examples, it should run, although some inputs might need tweaking like a project ID or something like that.

```py
def run(bootstrap_servers, topic, pipeline_args):
  # bootstrap_servers = '...'
  # topic = 'kafka_taxirides_realtime'
  # pipeline_args = ['--project', 'my-project', ...]
  ...

if __name__ == '__main__':
  import argparse

  parser = argparse.ArgumentParser()
  ...
  args, pipeline_args = parser.parse_known_args()

  run(args.bootstrap_servers, args.topic, pipeline_args)
```